chapters/data-processing.tex

```diff
@@ -0,0 +1,725 @@
+Note that while modern survey tools create unique identifiers for each data submission,
+that is not the same as having a unique ID variable for each observation in the sample,
+as there can be multiple submissions for the same observation.
+The unique identifier will be used to link data on observations in one dataset 
+with data on the same observations in other datasets, 
+such as other rounds of data collection and different data sources.
+Therefore, you must keep a canonical dataset of all observations in your data.
+This \textbf{master dataset}\sidenote{\url{
+		https://dimewiki.worldbank.org/Master\_Data\_Set}}
+is discussed in detail in Chapter 3, 
+and contains a record of all units ever encountered for each relevant level of observation,
+as well as their corresponding unique identifiers.
+
+
+% Merchand
+Checking for duplicated entries and making sure that each observation has a unique identifier are usually part of data quality monitoring.
+The lack of a unique identifier should be addressed as soon as data is received,
+so you can keep track of observations.
+\texttt{ieduplicates} and \texttt{iecompdup},
+two Stata commands included in the \texttt{iefieldkit}
+package\index{iefieldkit},\sidenote{\url{https://dimewiki.worldbank.org/iefieldkit}}
+create an automated workflow to identify, correct and document
+occurrences of duplicate entries.
+One advantage of using \texttt{ieduplicates}\sidenote{\url{
+		https://dimewiki.worldbank.org/wiki/ieduplicates}} 
+to correct duplicated entries is that it creates \textit{duplicates reports}
+that record which corrections made and why.
+Even if you are not using this command,
+it is important to include similar records in the \texttt{Documentation} folder.
+
+\subsection{Tidying raw data}
+
+Though raw data can be acquired in all shapes and sizes,
+development data is most commonly transferred as one or multiple tables.\sidenote{\textbf{Table:}
+	a two-dimensional dataset formed by rows and columns. 
+	Also called ``rectangular data''.
+	Examples of non-rectangular data are a text, or files, such as images.}
+Similarly, these tables can organize information in multiple ways,
+and not all of them result in easy-to-handle datasets.
+Fortunately, a vast literature of database management has identified the format
+that makes interacting with the data as easy as it can be.
+When applied to statistics, we call data in such format \textbf{tidy}.\sidenote{\textbf{Tidy data:}
+	Datasets where each column represents one variable,
+	each row represents one observation, 
+	and all variables in a table have the same unit of observation.}
+A dataset is tidy when each column represents one variable,\sidenote{\textbf{Variable:} 
+	the collection of all data points that measure the same attribute.}
+each row represents one observation, 
+and all variables in a table have the same unit of observation.
+Every other format is \textit{untidy} or just \textit{messy}.
```

Comment:
```suggestion
Every other format is \textit{untidy}.
```
I think it is confusing to use both terms, and I think _messy_ is too normative. A well organized wide dataset is objectively not messy, it is still untidy tho. If messy is a common term, then maybe say "Every other format is \textit{untidy} (sometimes also called \textit{messy})." so that we indicate which one is preferred.